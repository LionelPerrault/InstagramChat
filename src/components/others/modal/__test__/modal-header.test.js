import React from 'react'
import ModalHeader from '../modal-header'
import { create } from 'react-test-renderer'

describe('ModalHeader Component', () => {
  it('should match snapshot', () => {
    const tree = create(<ModalHeader title="Post likes" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
