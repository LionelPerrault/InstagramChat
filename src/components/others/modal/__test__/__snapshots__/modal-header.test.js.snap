// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalHeader Component should match snapshot 1`] = `
<div
  className="modal_header"
>
  <span
    className="title"
  >
    Post likes
  </span>
</div>
`;
